tests/unit/developers-dao.test.js

```diff
@@ -0,0 +1,167 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiExclude = require('chai-exclude');
+const config = require('config');
+const _ = require('lodash');
+const sinon = require('sinon');
+
+sinon.replace(config, 'get', () => ({ oracledb: {} }));
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+const developersDao = appRoot.require('api/v1/db/oracledb/developers-dao');
+const developersSerializer = appRoot.require('api/v1/serializers/developers-serializer');
+
+chai.should();
+chai.use(chaiExclude);
+chai.use(chaiAsPromised);
+
+describe('Test developers-dao', () => {
+  const fakeId = 'fakeId';
+
+  beforeEach(() => {
+    sinon.stub(conn, 'getConnection').resolves({
+      execute: (sql, sqlParams) => {
+        const sqlResults = {
+          multiResults: {
+            rows: [{}, {}],
+            outBinds: {
+              outId: [1],
+            },
+          },
+          singleResult: {
+            rows: [{}],
+            outBinds: {
+              outId: [1],
+            },
+          },
+          emptyResult: { rows: [] },
+        };
+        let result;
+        if (sql.includes('DELETE')) {
+          result = sqlResults.emptyResult;
+        } else if ('developerId' in sqlParams || 'id' in sqlParams) {
```

Comment:
Here is why we always suggest naming the variable be more descriptive. `id` here can stand for either a "reviewId" or a "gameId", which is confusing. Can you rename them to be more clear?